cpp/src/arrow/compute/kernels/scalar_temporal_test.cc

```diff
@@ -610,6 +673,7 @@ TEST_F(ScalarTemporalTest, TestNonexistentTimezone) {
     ASSERT_RAISES(Invalid, Year(timestamp_array));
     ASSERT_RAISES(Invalid, Month(timestamp_array));
     ASSERT_RAISES(Invalid, Day(timestamp_array));
+    ASSERT_RAISES(Invalid, YearMonthDay(timestamp_array));
```

Comment:
Should we also add tests with a valid timezone?

Reply:
I think those are already there in the TestZoned1 and TestZoned2 tests (?)

Reply:
Ah yes, my bad, I didn't look carefully at the diff context. Thanks.